.github/workflows/docker-and-ks8.yml

```diff
@@ -0,0 +1,128 @@
+name: Build agent packages.
+
+on: [push]
+
+env:
+  DOCKER_BUILDKIT: 1
+
+
+jobs:
+  # TODO move here tests from the CicleCi.
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image-type: [docker-json, docker-syslog, docker-api, k8s]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install python.
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8.10
+
+      - name: Perform the deployment of the test environment.
+        uses: ./.github/actions/perform-deployment
+        with:
+          deployment-name: "test_environment"
+
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
```

Comment:
Do we want to pin this image to a specific tag to avoid surprises and ensure better reproducibility?